### PR TITLE
[AC-5122] Initial changes for New Relic support

### PR DESCRIPTION
**Pull request by mc-nathan**
Work done:
- Add New Relic config file and initialization logic.

To test:
- Bang on https://staging.api.masschallenge.org and see the results here https://rpm.newrelic.com/accounts/1800634/applications/98015695

### CODE REVIEW

File: web/impact/impact/wsgi.py [hunk 1]
```diff
@@ -2,10 +2,18 @@
...
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'impact.settings')
+
 # environ must be set before importing jazzband/django-configurations
 from configurations.wsgi import get_wsgi_application  # noqa: E402
 
-
 application = get_wsgi_application()
+newrelic_application = newrelic.agent.wsgi_application()
+application = newrelic_application(get_wsgi_application())
```
Comment (shanbady) [inline web/impact/impact/wsgi.py]: remove the other call to ```application = get_wsgi_application()```

File: web/impact/newrelic.ini [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+# proxy_user =
+# proxy_pass =
+
+# Capturing request parameters is off by default. To enable the
+# capturing of request parameters, first ensure that the setting
+# "attributes.enabled" is set to "true" (the default value), and
+# then add "request.parameters.*" to the "attributes.include"
+# setting. For details about attributes configuration, please
+# consult the documentation.
+# attributes.include = request.parameters.*
```
Comment (shanbady) [inline web/impact/newrelic.ini:114]: we should consider enabling this

**Review by shanbady — CHANGES_REQUESTED** [review]

File: web/impact/newrelic.ini [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+# ---------------------------------------------------------------------------
+
+# Here are the settings that are common to all environments.
+
+[newrelic]
+
+# You must specify the license key associated with your New
+# Relic account. This key binds the Python Agent's data to your
+# account in the New Relic service.
+license_key = ba43edf69df19bdefa7f22fa72234276786d36f2
```
Comment (yotammanor) [inline web/impact/newrelic.ini:30]: Just checking - Is this a secret?

File: web/impact/requirements/base.txt [hunk 4]
```diff
@@ -8,6 +8,7 @@ hiredis==0.2.0
 markdown # Markdown support for the browsable API.
 mysqlclient
 oauthlib
+newrelic
```
Comment (yotammanor) [inline web/impact/requirements/base.txt:11]: I'd consider locking it to the version that accelerate auto-locks it, but not a blocker.

**Review by yotammanor — CHANGES_REQUESTED** [review]
A question and a non blocking comment. @shanbady's request is the blocking one.

File: web/impact/impact/wsgi.py (hunk 1 above)
Comment (mc-nathan) [reply web/impact/impact/wsgi.py]: Good catch.  Fixed.

File: web/impact/newrelic.ini (hunk 2 above)
Comment (mc-nathan) [reply web/impact/newrelic.ini:114]: I'd rather not sweat this at the moment.  I can see arguments on either side, so I lean towards going with the default until we need it.  I'd really like to get this out.

File: web/impact/newrelic.ini (hunk 3 above)
Comment (mc-nathan) [reply web/impact/newrelic.ini:30]: Hmm, probably should be.  Oof.

File: web/impact/requirements/base.txt (hunk 4 above)
Comment (mc-nathan) [reply web/impact/requirements/base.txt:11]: Won't fix.

File: web/impact/newrelic.ini (hunk 3 above)
Comment (mc-nathan) [reply web/impact/newrelic.ini:30]: Created follow on ticket to resolve this: AC-5204

**Review by shanbady — APPROVED** [review]

**Review by yotammanor — APPROVED** [review]

**Pull request merged**